### PR TITLE
edited14-joel8800.md

### LINKED ISSUE #3: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @ACaspeer
- [ ] @AndreiTeradyne
- [ ] @Daduc38
- [ ] @ElMT2023
- [ ] @JGT90
- [ ] @JameHann
- [ ] @JoelUsername
- [ ] @Raccoon137
- [ ] @frehiwotberhe
- [ ] @jgtres
- [ ] @joel8800
- [ ] @kbushe
- [ ] @mccroryn
- [ ] @rbelleau88
- [x] @shtaudor

### PR DESCRIPTION
**Pull request by joel8800**
I've updated 14-joel8800.md

## Updated 14-joel880.md
This resolves #3

### CODE REVIEW

File: _slides/14-joel8800.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I've added a new caption!
```
Comment (JoelUsername) [inline _slides/14-joel8800.md:6]: That's a great job. Thanks for this change!

Comment (jgtres) [inline _slides/14-joel8800.md:6]: it looks ok for me

**Review by JoelUsername — APPROVED** [review]

**Review by jgtres — APPROVED** [review]
Approving this

File: _slides/14-joel8800.md (hunk 1 above)
Comment (joel8800) [reply _slides/14-joel8800.md:6]: Thanks!

Comment (joel8800) [reply _slides/14-joel8800.md:6]: Thanks!

**Pull request merged**